<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginForm</class>
 <widget class="QWidget" name="LoginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WELCOME</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>uit.ico</normaloff>uit.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white
</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>171</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>uit.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>315</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>user.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>390</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>pass.png</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineUsername">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>350</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white; border-style: solid; border-width: 2px; border-radius: 15px; border-color: rgb(18,153,175); padding: 4px; color: #474747;</string>
   </property>
   <property name="text">
    <string>username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePassword">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>390</y>
     <width>350</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white; border-style: solid; border-width: 2px; border-radius: 15px; border-color: rgb(18,153,175); padding: 4px; color: #474747;</string>
   </property>
   <property name="text">
    <string>19522307</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonLogin">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>470</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #296091; border-style: outset; border-width: 2px; border-radius: 15px; border-color: white; padding: 4px; color:white</string>
   </property>
   <property name="text">
    <string>LOGIN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #296091;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="brandname">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>620</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>Made by 54010n</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>150</y>
     <width>421</width>
     <height>391</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>pictureLogin.png</pixmap>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
